Enterprise-CMCS/macpro-onemac
# Update sub sub email language

Story: https://qmacbis.atlassian.net/browse/OY2-29760
Endpoint:https://d1xwr2frq98eh7.cloudfront.net/
### Details
Minor Language change in sub sub emails to match: [Email Notification Confluence](https://qmacbis.atlassian.net/wiki/spaces/MACPRO/pages/3286138882/Email+Notifications+for+Package+Actions#Initial-Medicaid-SPA-Submission)

### Screenshots
State:
<img width="1107" alt="Screenshot 2024-09-03 at 10 26 33 AM" src="https://github.com/user-attachments/assets/d7bae7df-ac33-464f-9e21-091a8c6cea3f">

CMS:
<img width="1278" alt="Screenshot 2024-09-03 at 10 29 23 AM" src="https://github.com/user-attachments/assets/0fca552e-076a-4002-b1f0-be81a2656d58">


### Test Plan
1. Login as test user
2. Submit a sub sub action on any package which the action is available
3. Check CMS email notification to ensure it matches the confluence & ticket
4. Check the State email notification to ensure it also matches confluence & ticket
